docs/api-docs/store-management/shipping/use-a-shipping-provider.md

```diff
@@ -0,0 +1,375 @@
+# Use a Third-Party Shipping Provider
+
+Shipping provider apps (also known as carriers) allow merchants and shoppers to obtain **real-time** shipping quotes. Once a merchant installs a shipping provider app on their store, API users can connect the carrier to the store. API users can then define and enable multiple shipping methods for a carrier for any shipping zone. This makes real-time shipping methods available to shoppers for those zones during checkout.
```

Comment:
> API users can connect the carrier to the store. API users can then define and enable multiple shipping methods for a carrier for any shipping zone.

Does it make sense to call out that control panel users can also take these actions? i.e. "API users and control panel users can..."

Reply:
I have added merchants to the description